File: SEM 3/OOPL/LAB10/q1.java
import java.util.*;

class q1
{
    static <T> void swap(T arr[],T el1,T el2)
    {
        T temp;
        int pos1=0,pos2=0;
        for(int i=0;i<arr.length;i++)
        {
            if(arr[i]==el1)
            {
                pos1=i;
            }
            if(arr[i]==el2)
            {
                pos2=i;
            }
        }
        temp=arr[pos1];
        arr[pos1]=arr[pos2];
        arr[pos2]=temp;
    }
    public static void main(String[] args)
    {
        Scanner sc = new Scanner(System.in);
        int n;
        System.out.println("Enter Number of Elements : ");
        n=sc.nextInt();
        Integer arr[] = new Integer[n];
        for(int i=0;i<n;i++)
        {
            System.out.println("Enter Element "+(i+1));
            arr[i]=sc.nextInt();
        }
        System.out.println("Enter Elements to Swap");
        int el1=sc.nextInt();
        int el2=sc.nextInt();
        swap(arr, el1, el2);
        System.out.println();
        for(int i=0;i<n;i++)
        {
            System.out.print(arr[i]+" ");
        }
    }
}

File: SEM 4/DAAL/LAB 3/q1.c
#include <stdio.h>

void main()
{
    int n,temp,opcount=0;
    printf("Enter No. of Elements : \t");
    scanf("%d",&n);
    int arr[n];
    printf("Enter Array Elements : \n");
    for(int i=0;i<n;i++)
    {
        printf("Enter Element %d : \t",i+1);
        scanf("%d",&arr[i]);
    }
    for(int i=0;i<n-1;i++)
    {
        for(int j=i;j<n-1;j++)
        {
            opcount++;
            if(arr[j]>arr[j+1])
            {
                temp=arr[j];
                arr[j]=arr[j+1];
                arr[j+1]=temp;
            }
        }
    }
    printf("Operation Count is %d\n",opcount);
}